### PR TITLE
FAT-375232 Create new holdings for existing location (Thunderjet)

### PR DESCRIPTION
**Pull request by dnovikau**
[C375232](https://foliotest.testrail.io/index.php?/cases/view/375232)
***
[Allure report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/4391/allure/)
***

### CODE REVIEW

File: cypress/e2e/orders/inventory-interaction/create-new-holdings-for-existing-location.cy.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    Locations.deleteViaApi(testData.location);
+    ServicePoints.deleteViaApi(testData.servicePoint.id);
+    Users.deleteViaApi(testData.user.userId);
+  });
+
+  it(
+    'C375232 Create new holdings for already existing location when creating an order line (thunderjet)',
+    { tags: [TestTypes.extendedPath, DevTeams.thunderjet] },
+    () => {
+      const pol = {
```
Comment (IhorBohdan) [inline cypress/e2e/orders/inventory-interaction/create-new-holdings-for-existing-location.cy.js:103]: move data from it if it's possible

File: cypress/e2e/orders/inventory-interaction/create-new-holdings-for-existing-location.cy.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+        title: testData.instance.instanceTitle,
+        acquisitionMethod: ACQUISITION_METHOD_NAMES_IN_PROFILE.PURCHASE_AT_VENDOR_SYSTEM,
+        orderFormat: ORDER_FORMAT_NAMES.PHYSICAL_RESOURCE,
+        quantity: '1',
+        price: '20',
+        materialType: MATERIAL_TYPE_NAMES.BOOK,
+        inventory: 'Instance, holdings, item',
+      };
+
+      // Click on "PO number" link on "Orders" pane
```
Comment (IhorBohdan) [inline cypress/e2e/orders/inventory-interaction/create-new-holdings-for-existing-location.cy.js:113]: I think that comment doesn't need here

File: cypress/e2e/orders/inventory-interaction/create-new-holdings-for-existing-location.cy.js [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+                  });
+                });
+              },
+            );
+          });
+        });
+      });
+
+    cy.createTempUser([
+      // extra perms
```
Comment (IhorBohdan) [inline cypress/e2e/orders/inventory-interaction/create-new-holdings-for-existing-location.cy.js:67]: What is 'extra perms' what does it mean 'perms from test'?

File: cypress/e2e/orders/inventory-interaction/create-new-holdings.cy.js [hunk 4]
```diff
@@ -12,163 +10,160 @@ import NewOrganization from '../../../support/fragments/organizations/newOrganiz
...
   const barcodeForFirstItem = Helper.getRandomBarcode();
   const barcodeForSecondItem = Helper.getRandomBarcode();
 
-  let orderNumber;
-  let user;
-  let effectiveLocationServicePoint;
-  let firstLocation;
-  let instanceRecord = null;
-  let secondLocation;
+  const testData = {
```
Comment (IhorBohdan) [inline cypress/e2e/orders/inventory-interaction/create-new-holdings.cy.js:26]: Sorry, but why you fix my test?and remove some methods???

File: cypress/support/fragments/orders/orderLines.js [hunk 5]
```diff
@@ -30,15 +31,15 @@ import {
   ORDER_PAYMENT_STATUS,
 } from '../../constants';
 import InteractorsTools from '../../utils/interactorsTools';
+import selectLocationModal from './selectLocationModal';
```
Comment (IhorBohdan) [inline cypress/support/fragments/orders/orderLines.js:35]: Fix linter issues

File: cypress/e2e/orders/inventory-interaction/create-new-holdings-for-existing-location.cy.js (hunk 3 above)
Comment (IhorBohdan) [reply cypress/e2e/orders/inventory-interaction/create-new-holdings-for-existing-location.cy.js]: and why you're log in by admin?

File: cypress/support/fragments/orders/orderLines.js (hunk 5 above)
Comment (dnovikau) [reply cypress/support/fragments/orders/orderLines.js:34]: Fixed

File: cypress/e2e/orders/inventory-interaction/create-new-holdings.cy.js (hunk 4 above)
Comment (dnovikau) [reply cypress/e2e/orders/inventory-interaction/create-new-holdings.cy.js:26]: Fixed

File: cypress/e2e/orders/inventory-interaction/create-new-holdings-for-existing-location.cy.js (hunk 3 above)
Comment (dnovikau) [reply cypress/e2e/orders/inventory-interaction/create-new-holdings-for-existing-location.cy.js]: Fixed

File: cypress/e2e/orders/inventory-interaction/create-new-holdings-for-existing-location.cy.js (hunk 1 above)
Comment (dnovikau) [reply cypress/e2e/orders/inventory-interaction/create-new-holdings-for-existing-location.cy.js:94]: Fixed

File: cypress/e2e/orders/inventory-interaction/create-new-holdings-for-existing-location.cy.js (hunk 2 above)
Comment (dnovikau) [reply cypress/e2e/orders/inventory-interaction/create-new-holdings-for-existing-location.cy.js:94]: The comment left to make it easier to understand what test is actually doing. It's a step from related test case

Comment (IhorBohdan) [reply cypress/e2e/orders/inventory-interaction/create-new-holdings-for-existing-location.cy.js:94]: I think the test should be written in such a way that you can understand what's going on here without comment.
The best way to use comments is to use them for some part of the methods.

**Review by IhorBohdan — APPROVED** [review]

**Review by ostapwd — APPROVED** [review]

**Pull request merged**